lib/fourchette/heroku.rb

```diff
@@ -30,6 +30,12 @@ def client
     @heroku_client ||= PlatformAPI.connect(api_key)
   end
 
+  def legacy_client
+    api_key = ENV['FOURCHETTE_HEROKU_API_KEY']
+    ENV['HEROKU_API_KEY'] = api_key # necessary for Heroku::Auth.password to work
```

Comment:
Oh, nevermind, it's used... :P

but why is the `ENV['HEROKU_API_KEY']` required is still a valid question.